app/views/collections/form.html.erb

```diff
@@ -14,7 +14,7 @@
     <% end %>
   <% end %>
   <%= render Elements::HeadingComponent.new(level: :h1, variant: :h2, text: @collection_form.title || 'Untitled collection', classes: 'mb-3') %>
-  <%= render Elements::AlertComponent.new(variant: :danger, value: t('messages.validation')) if @collection_form.errors.present? %>
```

Comment:
I wonder if we might want to ticket investigating how our nested attrs implemention could make this better. For instance, if there are errors in a nested form (like the contributor form), should the errors be propagated to the top-level form (the work or collection)? If this is how AR errors would work with nested attrs (I can't recall), it may be worth the time the next time we touch H3.

Reply:
In spent some time on this and came to the conclusion that this is the same behavior as AR. However, I agree it could take more investigation because this feels awkward.